src/reynir_correct/errfinder.py

```diff
@@ -792,35 +861,35 @@ def VillaManns(self, txt: str, variants: str, node: Node) -> AnnotationDict:
         # Find the verb
         subjtree: Optional[SimpleTree] = self._simple_tree(ch0node)
         if subjtree is None:
-            return dict()
+            return None
         if self._sent.tree is None:
-            return dict()
-        subjwhole = self._sent.tree.first_match("NP-SUBJ >> { \"manns\" }")
+            return None
+        subjwhole = self._sent.tree.first_match('NP-SUBJ >> { "manns" }')
         if subjwhole is None:
-            return dict()
-        ip = subjwhole.enclosing_tag("IP")            
+            return None
+        ip = subjwhole.enclosing_tag("IP")
         if ip is None:
-            return dict()
+            return None
         so = ip.first_match("so_et")
         if so is None:
-            return dict()
+            return None
         start, end = so.span
         vars = set(so.all_variants) - {"et"}
         vars.add("ft")
         suggest = PatternMatcher.get_wordform(so.text.lower(), so.lemma, so.cat, vars)
-        return dict(
+        return AnnotationDict(
             text=f"Sögn á að vera í fleirtölu líkt og frumlagið '{subject}'",
             detail=f"Sögnin á að vera í fleirtölu, þrátt fyrir 'manns' í frumlagi.",
             start=start,
             end=end,
-            origin=so.tidy_text,
+            original=so.tidy_text,
```

Comment:
Here the TypedDict saved the day, and found a bug!

Reply:
Definitely better!